### ISSUE TITLE
free(): invalid next size (fast) error in testing 

### ISSUE THREAD

**Issue by oblivioncat** [AlexeyAB/darknet#627]
I have trained the model in my own training images and labels. When I loaded the latest .weights and test in a image(.png), a error happend:
Error in `./darknet': free(): invalid next size (fast): 0x00000000021031c0 ***
I compiled it in a docker container and run it on a GPU of 1080ti. Now one question is that if I compile it with GPU=0, 'test' command works well. But if it was run on a GPU the error happend.
More traceback attached here.
*** Error in `./darknet': free(): invalid next size (fast): 0x00000000021031c0 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7ff39c3d87e5]
/lib/x86_64-linux-gnu/libc.so.6(+0x8037a)[0x7ff39c3e137a]
/lib/x86_64-linux-gnu/libc.so.6(cfree+0x4c)[0x7ff39c3e553c]
./darknet[0x4472b1]
./darknet[0x46f715]
./darknet[0x46fcb5]
./darknet[0x454129]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0)[0x7ff39c381830]
./darknet[0x4051a9]
======= Memory map: ========
00400000-005d6000 r-xp 00000000 08:11 58006387                           /home/darknet/darknet
007d6000-007d7000 r--p 001d6000 08:11 58006387                           /home/darknet/darknet
007d7000-007d9000 rw-p 001d7000 08:11 58006387                           /home/darknet/darknet
007d9000-007da000 rw-p 00000000 00:00 0 
01dda000-0fff1000 rw-p 00000000 00:00 0                                  [heap]
10000000-10001000 rw-s 00000000 00:06 428                                /dev/nvidia0
10001000-10002000 rw-s 00000000 00:06 428                                /dev/nvidia0
10002000-10003000 rw-s 00000000 00:06 428                                /dev/nvidia0
10003000-10004000 rw-s 00000000 00:06 428                                /dev/nvidia0
10004000-10005000 rw-s 00000000 00:06 428                                /dev/nvidia0
......(omiited, since the text is so long)
7ff39bcc6000-7ff39bcc7000 r--p 00037000 08:11 11766415                   /usr/lib/x86_64-linux-gnu/libtbb.so.2
7ff39bcc7000-7ff39bcc9000 rw-p 00038000 08:11 11766415                   /usr/lib/x86_64-linux-gnu/libtbb.so.2
7ff39bcc9000-7ff39bccc000 rw-p 00000000 00:00 0 
7ff39bccc000-7ff39bd3a000 r-xp 00000000 08:11 11766509                   /usr/lib/x86_64-linux-gnu/mesa/libGL.so.1.2.0
Aborted (core dumped)

**Comment by oblivioncat** [comment]
Just solved this issue by replacing all of 'classes' and 'filter' in 'yolov3-voc.cfg' to new values(for me they are 1 and 18), since I just corrected only one pair of  'classes' and 'filter' values before(There are 3 pairs actually).

**Comment by AlexeyAB** [comment]
@oblivioncat Yes, everything should be done as described here: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects
